packages/grid/src/Grid.tsx

```diff
@@ -226,19 +227,26 @@ const Grid = ({
             expandLevel={ expandLevel }
             theme={ themeRef.current }
           />
-          { column.isExpandable && mappedItems[rowIndex].children ? (
-            <ExpandButtonWrapper onClick={ handleExpand }>
-              { mappedItems[rowIndex].isExpand
-                ? <RemoveIcon />
-                : <AddIcon /> }
-            </ExpandButtonWrapper>
-          ) : null }
+
 
           <BodyCellContent
             theme={ themeRef.current }
             center={ !!column.center }
             selected={ isSelected }
           >
+
+            { column.isExpandable && mappedItems[rowIndex].children ? (
```

Comment:
You can write 'column.isExpandable && mappedItems[rowIndex].children && (...)' instead 'column.isExpandable && mappedItems[rowIndex].children ? (...) : null'